controllers/employeesController.js

```diff
@@ -0,0 +1,142 @@
+const { type } = require('express/lib/response');
+const { check } = require('prettier');
+const db = require('../models');
+
+function getAttributesFromRequest(req){
+	return {
+		name: req.body.name,
+		lastName: req.body.lastName,
+		position: req.body.position
+	}
+}
+function validateEmployee(req, res) {
+	if (attributesInRequest(req, res) && attributesInRequestDefined(req, res) && checkTypeOfAttributes(req, res)) {
```

Comment:
No need to use  `if` statement. Use functions as conditions as follow:
`return attributesInRequest(req, res) && attributesInRequestDefined(req, res) && checkTypeOfAttributes(req, res)`

Reply:
Resolved in 83d7de4ed11bca24c689d1fe577bc38c8e589539